resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/KubernetesFeatureConfigStep.scala

```diff
@@ -19,7 +19,35 @@ package org.apache.spark.deploy.k8s.features
 import io.fabric8.kubernetes.api.model.HasMetadata
 
 import org.apache.spark.annotation.{DeveloperApi, Unstable}
-import org.apache.spark.deploy.k8s.SparkPod
+import org.apache.spark.deploy.k8s.{KubernetesConf, KubernetesDriverConf, KubernetesExecutorConf, SparkPod}
+
+@Unstable
+@DeveloperApi
+trait KubernetesFeatureDriverConfigStep extends KubernetesFeatureConfigStep {
+
+  protected var driverConf: KubernetesDriverConf = _
```

Comment:
Looks like we have to let the `driverConf` as `var` to make sure it can be set in `def init`